CHANGELOG.md

```diff
@@ -3,16 +3,23 @@
 ## v0.19.0 
```

Comment:
I have a feeling we're a little bit too liberal with changes to changelog, namely the version/tag should only be added when we release. Before that the newest entry in the changelog should always be "unreleased".